.travis.yml

```diff
@@ -8,5 +8,6 @@ before_script:
   - "sh -e /etc/init.d/xvfb start"
   - sleep 3 # give xvfb some time to start
 script:
- - npm run coverage
- - codecov -f .coverage/coverage-final.json -f .coverage/lcov.info
+ - if [ "$TRAVIS_EVENT_TYPE" == "cron" ]; then npm run testManual; fi
+ - if [ "$TRAVIS_EVENT_TYPE" != "cron" ]; then npm run coverage; fi
+ - if [ "$TRAVIS_EVENT_TYPE" != "cron" ]; then codecov -f .coverage/coverage-final.json -f .coverage/lcov.info; fi
```

Comment:
How does travis deal with failures? Abort on first? If it keeps going, 12 and 13 should be merged like this:

- if [ "$TRAVIS_EVENT_TYPE" != "cron" ]; then npm run coverage && codecov -f .coverage/coverage-final.json -f .coverage/lcov.info; fi